hooks/shotgun_filters.py

```diff
@@ -203,3 +203,37 @@ def get_link_filters(self, sg_location, entity_type, context_project, context_us
         )
 
         return link_filters
+
+    def get_preset_filters(self, sg_location, tab_name, entity_type):
```

Comment:
Do we need the `sg_location` and `tab_name` parameters? They don't seem to be used
```suggestion
    def get_preset_filters(self, entity_type):
```

Reply:
I'm easy either way but I added them to give the end users more parameters for controlling what filters to show. The location gives us the current entity the panel is in which might be useful.
The tab name might be less important but could be useful for things like the publish file tabs. If you wanted a different set of filters for publish_history vs publish_downstream/upstream for example the tab name would be required.

Reply:
Ah right, didn't think about providing the user with additional options to help filter. I'm ok if you want to add them back